django/test/client.py

```diff
@@ -158,6 +154,10 @@ def __call__(self, environ):
             response.close()                    # will fire request_finished
             request_finished.connect(close_old_connections)
 
+        # Attach the originating request to the response so that it could be
+        # later retrieved.
+        response.wsgi_request = request
```

Comment:
@renskiy Is it because `response.wsgi_request` will make `response` not picklable?

Also, won't this line move bring any issues? (doesn't seem so)